src/app/common/templates/config-data.yaml

```diff
@@ -0,0 +1,77 @@
+LegacyStructs:
+    # List of structs for which we output a legacy definition in af-structs.h
+    # Ideally this list should become empty.
+    - CredentialStruct
```

Comment:
What is this one even still used for? Seems like we have an opportunity to make the list empty soon

Reply:
Used throughout the door lock example apps:
```
% git grep CredentialStruct examples | grep -v matter: | wc -l
      67
```

Yes, I would like to remove it, but have not had a chance to shave that particular yak.